guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc

```diff
@@ -0,0 +1,36 @@
+[id="variables-for-generation-of-conversion-data_{context}"]
+= Variables for Generation of Conversion Data
+
+Before you run the Ansible role to generate conversion data, configure values of the following required Ansible variables.
+
+{Project} imports most of the required Ansible variables from the `{ansible-namespace}.convert2rhel` role.
+However, some of the required variables are not imported.
+These variables are marked with an asterisk * in the table below.
+You must create those additional variables manually and assign them to the `{ansible-namespace}.convert2rhel` role.
+
+.Required variables for conversion
+[cols="20%,5%,20%",options="header"]
+|====
+| Name | Type | Intent and value
+| `{project-context}_server_url` * | string | URL of your {ProjectServer}
+| `{project-context}_username` * | string | Your admin login, such as `admin`
+| `{project-context}_password` * | string | Your admin password
+| `{project-context}_organization` * | string | Name of your organization
+| `{project-context}_rhel_wait_for_syncs` * | boolean | Must be set to `false`
+| `{project-context}_validate_certs` * | boolean | Set to `true` if you wish to enable certificate checks in Ansible. (default: `true`)
```

Comment:
Why is this marked with an asterisk? From the description above it sounds like it's required but IMHO this shouldn't be required.

Reply:
What I tried to say with the asterisk was this: If there isn't an asterisk, the user can just edit the variable because it was imported with the role. If there is an asterisk, the user has to create the variable first to configure it, because it wasn't imported with the role. I.e. the asterisk was supposed to say that the user has to use slightly different steps. But if it's confusing then I should probably just leave it out.

Reply:
Required/optional is given by the table in which the variable is mentioned.

Reply:
Is it really that confusing? :D

Reply:
Okay, reword might be a good idea.

I was told that they are not imported due to the way Satellite imports Ansible variables. I have no fraking idea what that means. However, I did create a BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2126168
Not sure if I assigned it to the correct people. Can you pls check and fix it if needed?